app/models/grda_warehouse/enrollment_change_history.rb

```diff
@@ -50,6 +50,7 @@ def self.attributes_for_client_on_date client:, date:
         updated_at: Time.now,
         version: 1,
       }
+      # FIXME: how would current_user ever be defined here?
```

Comment:
I don't understand this use of current_user. Possibly from 8c7e49b6942

Reply:
That sure looks like a nasty bug.  I can confirm, basically the rescue with an array is causing no data to be saved.  That should have been `User.setup_system_user` as it is later in the line.

Reply:
thanks, I'll fix it up and remove the dangerous rescues. We can see if it crashes in practice